packages/app/src/app/components/Preview/DevTools/Console/index.tsx

```diff
@@ -236,8 +275,12 @@ class ConsoleComponent extends React.Component<StyledProps> {
       searchKeywordsHasError = true;
     }
 
+    // We need to access  this.props.store.preferences.settings.toggleConsoleEnabled
+    // to trigger console enabled/disabled message
+    const isConsoleEnabled = extractToggleConsoleEnabled(this.props);
+
     return (
-      <Container>
+      <Container key={(isConsoleEnabled as unknown) as number}>
```

Comment:
@christianalfoni Would this approach be correct to monitor `this.props.store.preferences.settings.toggleConsoleEnabled` value set in the global store to cause a re-render?
_Idea from https://overmindjs.org/guides/beginner/07_connectingcomponents?view=react&typescript=true#connecting-components-state-objects_

Is there a better way to "monitor" the settings value change?